src/hotspot/share/oops/markWord.hpp

```diff
@@ -176,7 +176,7 @@ class markWord {
     return (BasicLock*) value();
   }
   bool has_monitor() const {
-    return ((value() & monitor_value) != 0);
+    return ((value() & lock_mask_in_place) == monitor_value);
```

Comment:
So I'm a bit confused:

src/hotspot/share/oops/markWord.hpp:

`static const uintptr_t monitor_value            = 2;`

In the bug report you say:
> monitor value is 0b10

but that not what I'm seeing. What am I missing?

Reply:
Binary (0b) 10 == decimal 2 ;-)